include/libcgroup/groups.h

```diff
@@ -650,6 +650,31 @@ int cgroup_get_controller_version(const char * const controller,
  */
 enum cg_setup_mode_t cgroup_setup_mode(void);
 
+/**
+ * Return the number of controllers for the specified cgroup in @c libcgroup
```

Comment:
I did not understand `in @c ` part, sorry if I am missing the obvious.

Reply:
No clue what it is either; looks like my keyboard had a bad day :)